homeassistant/components/tuya/number.py

```diff
@@ -303,6 +303,28 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Thermostat
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf45ld5l0t9
+    "wk": (
+        NumberEntityDescription(
+            key=DPCode.BACKLIGHT,
+            translation_key="backlight_brightness",
+            native_unit_of_measurement=PERCENTAGE,
```

Comment:
Why is the unit hardcoded in this case? Normally Tuya provides the UoM in their APIs, right?